src/tests/gobprepare/importers/test_api_importer.py

```diff
@@ -0,0 +1,178 @@
+import pytest
+from unittest import TestCase
+from unittest.mock import patch, call, ANY
+
+from gobprepare.config import (
```

Comment:
Can you add a test for an empty GOB, ie GOB API returns an empty list?

Reply:
goog one. will do.